examples/basic/tidb-cluster.yaml

```diff
@@ -12,12 +12,14 @@ spec:
     requests:
       storage: "1Gi"
     config: {}
+    storageClassName: local-storage
```

Comment:
Please note this attribute as we would use default storageClass in basic example.

Reply:
ok , I have wrote annotation about this